### PR TITLE
Closes #1643, #1644, and #1645. Customer fixes.

### LINKED ISSUE #1643: Only admin can un-claim lesson.

### PR DESCRIPTION
**Pull request by michaelDpierce**

### CODE REVIEW

File: app/assets/templates/scheduling/sub-requests/sub-request.html [hunk 1]
```diff
@@ -14,7 +14,7 @@ <h1 class='modal-header modal-title'>Sub Request</h1>
       </tr>
       <tr>
         <th>Shift</th>
-        <td>{{shiftDate}} @ {{shiftStartTime}}-{{shiftEndTime}}</td>
+        <td>{{shiftStartTime}}-{{shiftEndTime}}</td>
```
Comment (duffcodester) [inline app/assets/templates/scheduling/sub-requests/sub-request.html]: nice.

File: app/views/sub_requests/processed.html.erb [hunk 2]
```diff
@@ -41,7 +41,7 @@
       <tbody>
         <tr ng-repeat='request in filteredRequests = (subRequests
                       | orderBy:predicate.value:predicate.reverse
-                      | filter:{shift: {location: {id: predicate.locationFilter || undefined}}}
+                      | filter:{location_id: predicate.locationFilter || undefined}
```
Comment (duffcodester) [inline app/views/sub_requests/processed.html.erb]: do we have location id directly on the sub request?

Comment (michaelDpierce) [inline app/views/sub_requests/processed.html.erb]: Yeah, I believe we are directly setting it on the $scope with a _.each

Comment (michaelDpierce) [inline app/views/sub_requests/processed.html.erb]: Before I fixed this you could see requests per location but not at "All Locations".

Comment (duffcodester) [inline app/views/sub_requests/processed.html.erb]: perfect.

File: app/views/private_lesson_mailer/thank_you.html.erb [hunk 3]
```diff
@@ -1,18 +0,0 @@
-<!DOCTYPE html>
```
Comment (duffcodester) [inline app/views/private_lesson_mailer/thank_you.html.erb]: is this in another template now?

**Comment by duffcodester** [comment]
:+1:

File: app/views/private_lesson_mailer/thank_you.html.erb (hunk 3 above)
Comment (michaelDpierce) [inline app/views/private_lesson_mailer/thank_you.html.erb]: Yeah, we moved this into the DigiQuatics mailer.

**Pull request merged**